builder/solver.go

```diff
@@ -205,7 +205,7 @@ func (s *solver) newSolveOptMulti(ctx context.Context, eg *errgroup.Group, onIma
 		cacheExports = append(cacheExports, newCacheExportOpt(s.cacheExport, false))
 	}
 	if s.maxCacheExport != "" {
-		cacheExports = append(cacheExports, newCacheExportOpt(s.cacheExport, true))
+		cacheExports = append(cacheExports, newCacheExportOpt(s.maxCacheExport, true))
```

Comment:
In main.go we only set one of them:

https://github.com/earthly/earthly/blob/17feb937be8f295f0e094a09b6999c82ad3a6a40/cmd/earthly/main.go#L2846-L2851